hack/e2e.sh

```diff
@@ -109,6 +109,17 @@ function test_elasticsearchcluster() {
 
 test_elasticsearchcluster
 
+
+function test_logs() {
+    if kubectl logs deployments/nav-e2e-navigator-controller \
+            | grep '^E' \
+            | grep -v 'reflector.go:205'; then
+        fail_test "Unexpected errors in controller logs"
+    fi
+}
+
+test_logs
```

Comment:
As much as this does make it easier to read logs, I think it could also obscure information and make debugging tests harder, especially if the developer isn't aware that we filter logs out. I think the kubernetes `gubernator` (for displaying logs from builds) can handle this sort of thing fairly well, although not 100% sure.

I'd prefer to improve the way we view logs, instead of altering the logs we're producing if that makes sense.